### PR TITLE
doc: update versions for 1.3.0

### PR DESCRIPTION
**Pull request by b-gent**
Updating NCS and nrfxlib versions in doc.

Signed-off-by: Bartosz Gentkowski <bartosz.gentkowski@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/gs_installing.rst [hunk 1]
```diff
@@ -229,9 +229,9 @@ To clone the repositories, complete the following steps:
 #. Determine what revision of the |NCS| you want to work with.
    The recommended way is to work with a specific release.
 
-   * To work with a specific release, the revision is the corresponding tag (for example, ``v1.2.0``).
+   * To work with a specific release, the revision is the corresponding tag (for example, ``v1.3.0``).
      You can find the tag in the :ref:`release_notes` of the release.
-   * To work with a development tag, the revision is the corresponding tag (for example, ``v1.2.99-dev1``)
+   * To work with a development tag, the revision is the corresponding tag (for example, ``v1.3.99-dev1``)
```
Comment (b-gent) [inline doc/nrf/gs_installing.rst:234]: is this change ok @ru-fu ?

Comment (ru-fu) [reply doc/nrf/gs_installing.rst:234]: I think it might be clearer to leave it at a tag that actually exists - what do you think, @carlescufi ?

**Comment by ru-fu** [comment]
Is MCUboot v1.5.99 still correct? According to the readme in the MCUboot repo we're at 1.6.0-rc2.
@carlescufi ?

**Comment by ru-fu** [comment]
@b-gent Can you also update the links to SES to link to the correct version (4.52)? See the Jira task.

**Comment by b-gent** [comment]
Adding DNM for now as this should be merged right before the release.

**Comment by b-gent** [comment]
> @b-gent Can you also update the links to SES to link to the correct version (4.52)? See the Jira task.
> (No need to update old release notes.)

done now

**Comment by b-gent** [comment]
This PR now also updates the manifest for [nrfxlib PR #207](https://github.com/nrfconnect/sdk-nrfxlib/pull/207)

When this is ready to go, [nrfxlib PR #207](https://github.com/nrfconnect/sdk-nrfxlib/pull/207) must be merged first, then manifest here updated with actual commit hash, and then we can merge this one.

**Comment by carlescufi** [comment]
@b-gent please update to SHA:
3d0d9a5691024f87b04c4a69fdd223dfae1fd1c4

File: doc/nrf/gs_installing.rst (hunk 1 above)
Comment (carlescufi) [reply doc/nrf/gs_installing.rst:234]: yes, please leave the existing one

**Comment by carlescufi** [comment]
> Is MCUboot v1.5.99 still correct? According to the readme in the MCUboot repo we're at 1.6.0-rc2.
> @carlescufi ?

This was a mistake when we tagged the MCUBoot repo, yes. It was identified by @mbolivar-nordic or @rlubos can't remember. But where do you see a reference to 1.5.99 in this PR or elsewhere?

**Comment by ru-fu** [comment]
> This was a mistake when we tagged the MCUBoot repo, yes. It was identified by @mbolivar-nordic or @rlubos can't remember. But where do you see a reference to 1.5.99 in this PR or elsewhere?

It's in the doc output: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/mcuboot/wrapper.html
And can be changed in the mcuboot/conf.py file (in this repo).

File: doc/nrf/gs_installing.rst (hunk 1 above)
Comment (b-gent) [reply doc/nrf/gs_installing.rst:234]: reverted to v1.2.99-dev1

**Comment by b-gent** [comment]
> @b-gent please update to SHA:
> 3d0d9a5691024f87b04c4a69fdd223dfae1fd1c4

updated

**Comment by carlescufi** [comment]
@b-gent please update doc/nrf/mcuboot/conf.py to `v1.6.0-rc2`

File: west.yml [hunk 2]
```diff
@@ -97,7 +97,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: v1.3.0-rc3
+      revision: 3d0d9a5691024f87b04c4a69fdd223dfae1fd1c4
```
Comment (tejlmand) [inline west.yml:100]: I assume this will be updated when doing the final tagging.

**Pull request merged**